app/common/expressions/forms.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, Mapping, Sequence
+
+from flask_wtf import FlaskForm
+from govuk_frontend_wtf.wtforms_widgets import GovRadioInput, GovSubmitInput, GovTextInput
+from wtforms import IntegerField, RadioField, SubmitField
+from wtforms.validators import DataRequired, Optional
+
+from app.common.data.types import ManagedExpressions
+
+
+class AddNumberConditionForm(FlaskForm):
+    type = RadioField(
+        "Only show the question if the answer is",
+        choices=[(ManagedExpressions.GREATER_THAN, ManagedExpressions.GREATER_THAN.value)],
```

Comment:
Some kind of `enum_to_choices(EnumClass)` helper would be good for this soon enough.